src/Butler/App/TodoManager.hs

```diff
@@ -332,5 +420,33 @@ startTodoManager ctx = do
                                 atomically $ modifyMemoryVar appStateM $ \tm -> do
                                     setSelectedTask tm $ TaskId $ fromInteger $ toInteger taskId
                             Nothing -> pure ()
+                    "show-settings" -> do
+                        logInfo "Show settings" ["ev" .= ev]
+                        atomically $ modifyMemoryVar appStateM $ \tm -> do
+                            tm{todoSettingsShow = not tm.todoSettingsShow}
+                    "setting-show-column-prio" -> do
+                        logInfo "Show settings" ["ev" .= ev]
+                        atomically $ modifyMemoryVar appStateM $ \tm -> do
+                            tm
+                                { todoSettings =
+                                    tm.todoSettings
+                                        { showColumnPrio = not tm.todoSettings.showColumnPrio
+                                        }
+                                }
+                    "setting-show-column-dueDate" -> do
+                        logInfo "Show settings" ["ev" .= ev]
+                        atomically $ modifyMemoryVar appStateM $ \tm -> do
+                            tm
+                                { todoSettings =
+                                    tm.todoSettings
+                                        { showColumnDueDate = not tm.todoSettings.showColumnDueDate
+                                        }
+                                }
                     _ -> logError "Unknown trigger" ["ev" .= ev]
             ev -> logError "Unknown ev" ["ev" .= ev]
+
+parseDueDate :: MonadFail m => String -> m TaskDueDate
```

Comment:
Perhaps, instead of failing (and crashing the app), it might be better to silently return `NoDueDate` in the event the date can't be parsed.